b2/console_tool.py

```diff
@@ -3454,6 +3454,8 @@ class License(Command):  # pragma: no cover
             'https://raw.githubusercontent.com/PythonCharmers/python-future/master/LICENSE.txt',
         'pefile':
             'https://raw.githubusercontent.com/erocarrera/pefile/master/LICENSE',
+        'https://github.com/python/typeshed':
```

Comment:
Are we using the repository version? Why? Even their original readme says that you don't need to and points to:
```
pip install types-six types-requests
```
instead. Those these libraries provide no licensing information?
https://github.com/python/typeshed#using

Reply:
1) no, we aren't. This is URL extracted by pip-licenses. In fact this is not even a direct dependency of ours, so we don't specify it anywhere ourselves.
2) piplicense is unable to get LICENSE text for it. It is able to extract just license type (how it does it, idk, but I expect it to be in at least python trove classifiers for this package). It is not enough for our purpose, hence the direct link.